src/component/Options.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { createRef, useEffect, useState } from 'react';
+
+import { PROFILE_URL } from '../auth';
+import { getStorageItem, setStorageItem } from '../storage';
+import Box from '@mui/material/Box';
+import { TextField, Button, Link, Typography, Snackbar, Alert } from '@mui/material';
+import SettingsIcon from '@mui/icons-material/Settings';
+import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import LoginIcon from '@mui/icons-material/Login';
+import SaveAltIcon from '@mui/icons-material/SaveAlt';
+import CloseIcon from '@mui/icons-material/Close';
+import Divider from '@mui/material/Divider';
+import { v4 as uuidv4 } from 'uuid';
+
+const EditableList = () => {
+  const [text, setText] = useState('');
+  const [open, setOpen] = useState(false);
+  const [severity, setSeverity] = useState<'success' | 'error'>('success');
+  const [message, setMessage] = useState('');
+
+  useEffect(() => {
+    (async () => {
+      const allowList = (await getStorageItem('allowList')) ?? [];
+      setText(allowList.join('\n'));
+    })();
+  }, []);
+
+  return (
+    <>
+      <Typography variant="h6">Allow List</Typography>
```

Comment:
```suggestion
      <Typography variant="h6">Allowlist</Typography>
```
(Similar changes elsewhere.)